### DISCUSSION TITLE
Is helloSystem a serious project?

### DISCUSSION THREAD

**Discussion by hack3rcon** [helloSystem/hello#563] [Q&A] [answered]
Hello,
Why does helloSystem still not have a website after several years?

Thanks.

**Comment by probonopd** [comment, marked as answer]
Hi, yes, it is "serious" (in the sense that you can download and run it, and you can get its source code to make your own).

But it is a purely volunteer based, noncommercial community driven open source project, which means that people contribute to it whenever they have some spare time and want to contribute to it.

The project does have a website:

https://hellosystem.github.io/ is its official website.
